CMakeLists.txt

```diff
@@ -1,9 +1,6 @@
-# =============================================================================
-# Copyright (C) 2018-2022 Blue Brain Project
+# Copyright 2023 Blue Brain Project, EPFL. See the top-level LICENSE file for details.
```

Comment:
```suggestion
# Copyright 2023 Blue Brain Project, EPFL.
See the top-level LICENSE file for details.
```
**nitpick**: noticed this because this is the first file in review :)

Reply:
This is cmake-formatter that ask for it, so...